core/src/main/java/org/commonjava/maven/ext/core/groovy/ASTTransformer.java

```diff
@@ -59,17 +59,18 @@
 import static org.apache.groovy.ast.tools.ClassNodeUtils.addGeneratedMethod;
 
 /**
- * Ensures that Groovy scripts annotated with {@link } are transformed into a class that
- * extends {@link }.
+ * Ensures that Groovy scripts annotated with {@link GMEBaseScript} and {@link PMEBaseScript} are transformed into a class that
+ * extends groovy.transform.BaseScript.
```

Comment:
Did we want to keep the `@link` for this one?

Reply:
Yeah, good spot.